test/setup/deploy_svc.sh

```
@@ -0,0 +1,86 @@
+    step certificate create "maestro-grpc-broker-ca" ${grpc_broker_cert_dir}/ca.crt ${grpc_broker_cert_dir}/ca.key --profile root-ca --no-password --insecure
+    step certificate create "maestro-grpc-broker-server" ${grpc_broker_cert_dir}/server.crt ${grpc_broker_cert_dir}/server.key -san maestro-grpc-broker -san maestro-grpc-broker.maestro -san maestro-grpc-broker.maestro.svc -san localhost -san 127.0.0.1 --profile leaf --ca ${grpc_broker_cert_dir}/ca.crt --ca-key ${grpc_broker_cert_dir}/ca.key --no-password --insecure
+    cat << EOF > ${grpc_broker_cert_dir}/cert.tpl
+{
+    "subject":{"organization":"open-cluster-management","commonName":"grpc-client"},
+    "keyUsage":["digitalSignature"],
+    "extKeyUsage": ["serverAuth","clientAuth"]
+}
+EOF
+    step certificate create "maestro-grpc-broker-client" ${grpc_broker_cert_dir}/client.crt ${grpc_broker_cert_dir}/client.key --template ${grpc_broker_cert_dir}/cert.tpl --ca ${grpc_broker_cert_dir}/ca.crt --ca-key ${grpc_broker_cert_dir}/ca.key --no-password --insecure
+    kubectl create secret generic maestro-grpc-broker-cert -n $namespace --from-file=ca.crt=${grpc_broker_cert_dir}/ca.crt --from-file=server.crt=${grpc_broker_cert_dir}/server.crt --from-file=server.key=${grpc_broker_cert_dir}/server.key --from-file=client.crt=${grpc_broker_cert_dir}/client.crt --from-file=client.key=${grpc_broker_cert_dir}/client.key
+    kubectl create secret generic maestro-grpc-broker-cert -n $agent_namespace --from-file=ca.crt=${grpc_broker_cert_dir}/ca.crt --from-file=server.crt=${grpc_broker_cert_dir}/server.crt --from-file=server.key=${grpc_broker_cert_dir}/server.key --from-file=client.crt=${grpc_broker_cert_dir}/client.crt --from-file=client.key=${grpc_broker_cert_dir}/client.key
```

Shellcheck (0.11.0)

[warning] agent_namespace is referenced but not assigned.

(SC2154)